src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,67 @@
+package storage;
+
+import task.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    protected String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+
+    // copy file into TaskList
+    public ArrayList<Task> load() throws IOException {
+        ArrayList<Task> tasks = new ArrayList<>();
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String nextLine = s.nextLine();
+            String[] split = nextLine.split("/");
+            String indicator = split[0];
+            switch (indicator) {
+                case "D":
```

Comment:
According to the checkstyle coding standard, everything within this switch block should be indented 4 spaces to the left from where it currently is (ie. 'case' should be in line with 'switch')